### PR TITLE
chore: PPO system cleanup

### PR DESCRIPTION
**Pull request by sash-a**
## What?
Did a couple small clean ups to the PPO systems that are long overdue:
* Removed all the capital comments
* Removed opt states from the losses
* Removed tree map around the done reshape during acting (it's just an array)
* Removed metrics from the traj batch because they aren't really part of the batch
* Cleaner unpacking of the loss metrics

@RuanJohn should we add this for MAT?

### CODE REVIEW

File: mava/systems/mat/anakin/mat.py
Comment (RuanJohn) [inline mava/systems/mat/anakin/mat.py:120]: A slight concern I have with not storing info in the transition is that we lose the ability to quickly edit systems that could need it in the future. What do you think?

File: mava/systems/mat/anakin/mat.py
Comment (RuanJohn) [inline mava/systems/mat/anakin/mat.py]: If we change from `loss_actor` -> `actor_loss` can we please do the same for `loss_actor1` and `loss_actor2`

File: mava/systems/mat/anakin/mat.py
Comment (RuanJohn) [inline mava/systems/mat/anakin/mat.py:259]: Why do we no longer need to use this?

File: mava/systems/ppo/anakin/ff_ippo.py
Comment (RuanJohn) [inline mava/systems/ppo/anakin/ff_ippo.py]: If we change value -> critic I think we should be consistent with the naming everywhere. Are you keeping it as value to still be comparable with old experiments?

File: mava/systems/ppo/anakin/ff_ippo.py
Comment (RuanJohn) [inline mava/systems/ppo/anakin/ff_ippo.py]: No so sure about appending `raw_` to the loss values. I think `actor_loss` and `raw_actor_loss` are the same in this case as well. So we could always just catch one of them with a `_` and used the other.

File: mava/systems/ppo/anakin/ff_mappo.py
Comment (RuanJohn) [inline mava/systems/ppo/anakin/ff_mappo.py:92]: I like this!

File: mava/systems/ppo/anakin/ff_mappo.py
Comment (RuanJohn) [inline mava/systems/ppo/anakin/ff_mappo.py:153]: Long overdue!

**Review by RuanJohn — CHANGES_REQUESTED** [review]
Thank you @sash-a! I think these are some very needed changes. I just have a few suggestions and questions. Any comment applies to all systems.

**Comment by sash-a** [comment]
todo: same fixes for sable and switch sable to using the PPOTransition

File: README.md
Comment (WiemKhlifi) [inline README.md:187]: Are we giving up on this because it's no longer maintained?

Comment (sash-a) [reply README.md:187]: Ya @RuanJohn and I discussed it. I think if we want to bring it back at some point it would be nice, but right now it's not maintained and I'm not even sure if it works right now

Comment (WiemKhlifi) [reply README.md:187]: Indeed I agree on that it has some issues within the evaluator as well, hpefully we can restore it in future 🤞

**Review by OmaymaMahjoub — DISMISSED** [review]
LGTM! Let's just wait for the benchmark before we merge this 🙏

**Review by RuanJohn — DISMISSED** [review]
:trophy:

**Review by OmaymaMahjoub — APPROVED** [review]
Thanks @sash-a 🙌

**Review by RuanJohn — APPROVED** [review]
:star2:

**Pull request merged**
